lib/msprime.c

```diff
@@ -2219,42 +2224,42 @@ msp_alloc_pedigree(msp_t *self, size_t num_inds, size_t ploidy)
 
 /* TODO merge this method into where it's called - we don't really need these
  * arrays any more. */
-static int MSP_WARN_UNUSED
-msp_set_pedigree(msp_t *self, tsk_id_t *parents, double *times, tsk_flags_t *is_sample)
-{
-    size_t i, j;
-    tsk_id_t parent_ix;
-    tsk_flags_t sample_flag;
-    size_t sample_num;
-    individual_t *ind = NULL;
-
-    tsk_bug_assert(self->pedigree != NULL);
-
-    ind = self->pedigree->inds;
-    sample_num = 0;
-    for (i = 0; i < self->pedigree->num_inds; i++) {
-        ind = &self->pedigree->inds[i];
-        ind->id = (tsk_id_t) i;
-        ind->time = times[i];
-
-        // Link individuals to parents
-        for (j = 0; j < self->ploidy; j++) {
-            parent_ix = parents[i * self->ploidy + j];
-            if (parent_ix != TSK_NULL) {
-                ind->parents[j] = &self->pedigree->inds[parent_ix];
-            }
-        }
-        // Set samples
-        sample_flag = is_sample[i];
-        if (sample_flag != 0) {
-            tsk_bug_assert(sample_flag == 1);
-            self->pedigree->samples[sample_num] = ind;
-            sample_num++;
-        }
-    }
-    self->pedigree->num_samples = sample_num;
-    return 0;
-}
+// static int MSP_WARN_UNUSED
```

Comment:
This isn't really a minimal diff - the minimal change wouldn't touch this code at all